_shared/project/.github/workflows/ci.yml

```diff
@@ -1,6 +1,27 @@
 name: CI
 on:
   push:
+    paths-ignore:
+      - '.cookiecutter/*'
+      - '.github/dependabot.yml'
+      - '.github/workflows/deploy.yml'
+      - '.github/workflows/redeploy.yml'
+      - 'bin/logger'
+      - 'bin/make_python'
+      - 'bin/make_template'
+      - 'conf/development.ini'
+      - 'conf/production.ini'
+      - 'conf/supervisord*.conf'
+      - 'docs/*'
+      - 'requirements/*.in'
+      - 'requirements/dev.txt'
+      - '.docker.env'
+      - '**/.gitignore'
+      - '.python-version'
+      - 'Dockerfile'
+      - 'LICENSE'
+      - '*.md'
+      - 'docker-compose.yml'
```

Comment:
Some of the paths that are listed are quite low-value in that they're files that we don't update very often so they won't save us many workflow runs. Nonetheless, since we can make the change just once here in the cookiecutter and have it be automatically applied to all cookiecutter-using apps (and we can update this list of ignores in the same semi-automated way) I thought it worth going to the effort to list all the cookiecutter-generated paths that can be ignored.